src/services/puzzle/puzzle-formats.ts

```diff
@@ -0,0 +1,21 @@
+import { PuzzleItem } from "../../database/puzzle-db.js";
+
+export function isValidPuzzleItemFormat(obj: PuzzleItem): boolean {
+    if (typeof obj.userId !== "string") {
+        return false;
+    }
+
+    if (typeof obj.teamName !== "string") {
+        return false;
+    }
+
+    if (typeof obj.lastCorrect !== "number") {
+        return false;
+    }
```

Comment:
Use functions provided in `FormatTools.ts`

Reply:
Honestly might just get rid of this function because client never passes a puzzle item that we need to validate types for